pandas/io/parsers/readers.py

```diff
@@ -1515,6 +1515,12 @@ def _clean_options(
             if engine == "c" and sep == r"\s+":
                 result["delim_whitespace"] = True
                 del result["delimiter"]
+            elif engine == "pyarrow" and sep == r"\s+":
+                fallback_reason = (
+                    "the 'pyarrow' engine does not support separators "
+                    "with more than one character, except for '\\t'. Please "
```

Comment:
This is not 100% clear to me. Does this mean that `\t\t` is a valid separator for the pyarrow parser? Do we have a test for valid separators where this can be added if that's the case?

Reply:
I'm not 100% sure either, i should better change the error message to "the 'pyarrow' engine does not support the \s+ separator. Please use the C parser instead.". What do you think?